index.html

```diff
@@ -289,5 +289,61 @@ <h1 class="title">Skills</h1>
 
       <!-- ^^^^^^^^^^^^^ Category skills in about section ^^^^^^^^ -->
     </section>
+
+    <!-- -------- contact-section ---------- -->
+    <section class="contact-section">
+      <div class="contact">
+        <div class="contact__header">
+          <h1 class="header__title">Contact Me</h1>
+          <p class="header__text">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <form
+          class="contact-form"

```

Comment:
- [ optional] you can display your form flex to layout elements

Reply:
thanks, I fixed that.